.ado/templates/react-native-macos-init.yml

```diff
@@ -63,7 +63,7 @@ steps:
   - task: CmdLine@2
     displayName: Bump package version
     inputs:
-      script: node scripts/bump-oss-version.js --nightly
+      script: node scripts/bump-oss-version.js --ci 0.62.1000
```

Comment:
Does this mean we now have to update this every time we create a new stable branch or could this be inferred by the script from the current branch name?

Reply:
To bump the stable version of the `react-native-macos` package [we use a different script](https://github.com/microsoft/react-native-macos/blob/538a2677e8b7ed006eca49b79e3b2989962fe4d5/.ado/publish.yml#L55-L59). Would it be possible to use that here too and not introduce forked changes to the `bump-oss-version.js` script?